htdocs/core/class/fiscalyear.class.php

```diff
@@ -381,13 +381,19 @@ public function info($id)
 	 *	@param	int		$dateend	Date end to scan
 	 *	@return	string				Number of entries
 	 */
-	public function getAccountancyEntriesByFiscalYear($datestart, $dateend)
+	public function getAccountancyEntriesByFiscalYear($datestart = '', $dateend = '')
 	{
 		global $conf;
 
+		if(empty($datestart) )
+			$datestart = $this->date_start;
+		if(empty($dateend) )
+			$dateend = $this->date_end;
+
 		$sql = "SELECT count(DISTINCT piece_num) as nb";
 		$sql.= " FROM ".MAIN_DB_PREFIX."accounting_bookkeeping ";
 		$sql.= " WHERE doc_date >= '".$datestart."' and doc_date <= '".$dateend."'";
```

Comment:
Because $datestart and $dateend are timestamp, you must format then into sql date by using 
WHERE doc_date >= '".$this->db->idate($datestart)."' and doc_date <= '".$this->db->idate($dateend)."'";

Can you fix this ?